tiledb/sm/array_schema/dimension.cc

```diff
@@ -1219,7 +1242,7 @@ ByteVecValue Dimension::map_from_uint64<char>(
   if (ret.back() != 128)
     ret.push_back(128);
 
-  return ByteVecValue(std::move(ret));
+  return ByteVecValue(std::vector<uint8_t>{ret.begin(), ret.end()});
```

Comment:
I'm starting to think we shouldn't be making all of these needless copies just to track the memory usage for what appears to be a millisecond or so.